app/database.py

```diff
@@ -57,4 +57,29 @@ def createNewChallenge(creator_id, title, path, dt_submit, dt_finish):
         dt_finish=dt_finish,
         finished=False)
     db.session.add(newChallenge)
-    db.session.commit()
\ No newline at end of file
+    db.session.commit()
+
+# Given a Challenge model, find the submission with the fewest articles in its path (the winner of the challenge)
+def findWinner(challenge):
+    maxArticles = 51 # For what we currently have, each path has a maximum length of 50 articles
```

Comment:
Although maximum is currently 50, apart from the field list everything is set up to be dynamically scaled, theoretically beyond 50.

I think a better solution would be to set maxArticles to -1, and then in the below if statement add an "or maxArticles == -1"

Reply:
Done